examples/svdiff/requirements.txt

```diff
@@ -0,0 +1,7 @@
+accelerate>=0.16.0
+torchvision
+transformers>=4.25.1
+safetensors
+ftfy
+tensorboard
+Jinja2
```

Comment:
For logging to wandb, we need `wandb` as a dependency as well.

Reply:
fixed. https://github.com/huggingface/diffusers/pull/3107/commits/74e2a086b822cf16425dc058d92670f6d7a05142